### PR TITLE
Makes acceptance test more stable with Travis

### LINKED ISSUE #734: There are random failures with acceptance test when running in  CI + Chrome
In half of the test runs in CI, there is a failure. Most of the failures seems to be related on opening new window. The problem can not be reproduced when running tests on a local PC.

Think how to resolve the problem and take corrective actions.

### PR DESCRIPTION
**Pull request by aaltat**
There is quite often a random failures with test that open new window.
It seem that with Travis is takes little bit of time to open a window
and this fix wait the windows to open before test is performed.

Partially fixes #734

### CODE REVIEW

File: test/acceptance/keywords/navigation.robot [hunk 1]
```diff
@@ -101,6 +101,13 @@ Target Opens in New Window
...
+    Wait Until Keyword Succeeds    5    1    Wait Untill Window Is Open
     Select Window    ${INDEX TITLE}
     Verify Location Is "index.html"
     [Teardown]    Run Keyword If Test Passed    Run Keywords    Close Window    Select Window
+
+*** Keywords ***
+Wait Untill Window Is Open
+    ${titles} =    Get Window Titles
+    ${status} =    Evaluate    len(${titles}) > 1
+    Should Be True    ${status}
```
Comment (pekkaklarck) [inline test/acceptance/keywords/navigation.robot]: `Wait Until Window Is Open` sounds like a keyword that could be in the library itself. Also, there's only one `L` in `Until`.

**Comment by aaltat** [comment]
There has been some complaints from users too, example #750 and we should build such `Wait*` keyword for window selection too.

I will fix the typo later today.

**Comment by aaltat** [comment]
Created #757 to create the new keyword

File: test/acceptance/keywords/navigation.robot (hunk 1 above)
Comment (aaltat) [reply test/acceptance/keywords/navigation.robot]: Done

**Pull request merged**